modules/data-mover-for-csi-snapshots.adoc

```diff
@@ -0,0 +1,216 @@
++
+.Example Data Protection Application (DPA) CR
+[source,yaml]
+----
+apiVersion: oadp.openshift.io/v1alpha1
+kind: DataProtectionApplication
+metadata:
+  name: velero-sample
+  namespace: openshift-adp
+spec:
+  features:
+    dataMover:
+      enable: true
+      credentialName: <secret_name> <1>
+  backupLocations:
+    - velero:
+        config:
+          profile: default
+          region: us-east-1
+        credential:
+          key: cloud
+          name: cloud-credentials
+        default: true
+        objectStorage:
+          bucket: <bucket_name>
+          prefix: <bucket_prefix>
+        provider: aws
+  configuration:
+    restic:
+      enable: <true_or_false>
+    velero:
+      defaultPlugins:
+        - openshift
+        - aws
+        - csi
+----
+<1> Add the Restic secret name from the previous step. If this is not done, the default secret name `dm-credential` is used.
++
+The OADP operator installs two custom resource definitions (CRDs), `VolumeSnapshotBackup` and `VolumeSnapshotRestore`.
++
+.Example `VolumeSnapshotBackup` CRD+
+[source,yaml]
+----
+apiVersion: datamover.oadp.openshift.io/v1alpha1
+kind: VolumeSnapshotBackup
+metadata:
+  name: <vsb_name>
+  namespace: <namespace_name> <1>
+spec:
+  volumeSnapshotContent:
+    name: <snapcontent_name>
+  protectedNamespace: <adp_namespace>
+  resticSecretRef:
+    name: <restic_secret_name>
+----
+<1> Specify the namespace where the volume snapshot resides.
++
+.Example `VolumeSnapshotRestore` CRD+
+[source,yaml]
+----
+apiVersion: datamover.oadp.openshift.io/v1alpha1
+kind: VolumeSnapshotRestore
+metadata:
+  name: <vsr_name>
+  namespace: <namespace_name> <1>
+spec:
+  protectedNamespace: <protected-ns> <2>
+  resticSecretRef:
+    name: <restic_secret_name>
+  volumeSnapshotMoverBackupRef:
+    sourcePVCData:
+      name: <source_pvc_name>
+      size: <source_pvc_size>
+    resticrepository: <your_restic_repo>
+    volumeSnapshotClassName: <vsclass_name>
+----
+<1> Specify the namespace where the volume snapshot exists.
+<2> Specify the namespace where the Operator is installed. The default is `openshift-adp`.
+
+. You can back up a volume snapshot by performing the following steps:
+
+.. Create a backup CR:
++
+[source,yaml]
+----
+apiVersion: {velero-domain}/v1
+kind: Backup
+metadata:
+  name: <backup_name>
+  namespace: <protected_ns> <1>
+spec:
+  includedNamespaces:
+  - <app_ns>
+  storageLocation: velero-sample-1
+----
+<1> Specify the namespace where the Operator is installed. The default namespace is `openshift-adp`.
+
+.. Wait up to 10 minutes and check whether the `VolumeSnapshotBackup` CR status is `Completed` by entering the following commands:
+
++
+[source,terminal]
+----
+$`oc get vsb -n <app_ns>`
+----
++
+$`oc get vsb <vsb_name> -n <app_ns> -ojsonpath="{.status.phase}`
+----
+
++
+
++
+A snapshot is created in the object store that was specified in the Restic secret.
```

Comment:
```suggestion
+
A snapshot is created in the object store that was specified in the Restic secret.
```
You just need one `+ ` and no extra line breaks. Hopefully, when you fix the source block above, then this will render correctly.

![image](https://user-images.githubusercontent.com/82119979/187963645-905b2b2b-e524-4ad6-9665-d708ef4b966d.png)